pact/Pact/Core/IR/Term.hs

```diff
@@ -511,28 +520,30 @@ instance (Pretty name, Pretty builtin, Pretty ty) => Pretty (Term name ty builti
     prettyLamArg (Arg n ty _) =
       pretty n <> prettyTyAnn ty
 
+unSeqTerm :: Term name ty builtin info -> [Term name ty builtin info]
+unSeqTerm (Sequence e1 e2 _) = e1 : unSeqTerm e2
+unSeqTerm e = [e]
+
 instance (Pretty name, Pretty builtin, Pretty ty) => Pretty (TopLevel name ty builtin info) where
   pretty = \case
     TLTerm tm -> pretty tm
-    _ -> "todo: pretty defs/modules"
-
-prettyDef :: Pretty ty => Doc ann -> Arg ty info -> info -> [Arg ty info] -> Doc ann
-prettyDef deftoken (Arg defname defRTy _) dInfo defArgs =
-  let dfNameArg = Arg defname defRTy dInfo
-      argList = parens (hsep (pretty <$> defArgs))
-  in parens $ deftoken <+> pretty dfNameArg <+> argList
+    TLModule m -> pretty m
+    _ -> "todo: pretty interfaces"
```

Comment:
Is there ever an instance where we'll need to prettyprint an interface?

Reply:
Probably yes for debugging. Thanks for the note, I'll remember to add pretty instances.

For transparency: These `pretty` instances are being used during integration work to debug the pact-5 representation of the modules read in from legacy. They're incredibly useful to see how we are deserializing things, but only interfaces presented a challenge.